test/pdfs/bug1773680.pdf.link

```diff
@@ -0,0 +1,2 @@
+https://bugzilla.mozilla.org/attachment.cgi?id=9280675
+
```

Comment:
According to the test results and logs, this document is a duplicate of `issue12233.pdf` and shouldn't be added.

Hence the manifest-entry below should reference the existing file instead, assuming that we even need to add a new test here (since the existing test-cases may already cover this).